docs/source/cuda/types.rst

```diff
@@ -0,0 +1,49 @@
+Types on CUDA target
+====================
+
+Note: This page is about types specific to CUDA targets. Most types on CPU targets are also
+available in CUDA target, for information on these types, see :ref:`numba-types`.
```

Comment:
There is a list of Python / NumPy types supported in the CUDA target at https://numba--7949.org.readthedocs.build/en/7949/cuda/cudapysupported.html#built-in-types - could you update the reference to point there?